### ISSUE TITLE
The suggested wireless network configuration is problematic

### ISSUE THREAD

**Issue by myfreeweb** [t6/freebsd-runit#3]
Maybe I only noticed this because wpa_supplicant takes a significant amount of time for me (because I use an EAP-TLS secured network), but stuff like this:

```shell
ifconfig wlan0 destroy > /dev/null 2>&1 || true
```

in dhclient's conf does not work:

- dhclient/conf creates the interface
- dhclient starts waiting for link
- wpa_supplicant starts doing its thing
- dhclient itself times out, restarting the service
- which executes the conf again, which destroys the interface
- wpa_supplicant is interrupted by the destruction (so we never actually connect to the network)
- GOTO 0

**Comment by t6** [comment]
Yeah, the suggested wireless networking setup is not ideal. I'm open to suggestions.
